keras/legacy/models.py

```diff
@@ -528,6 +530,8 @@ def test_on_batch(self, data, sample_weight={}, **kwargs):
 
     def predict_on_batch(self, data):
         output_list = super(Graph, self).predict_on_batch(data)
+        if not isinstance(output_list,list):
```

Comment:
ya it was a copy-paste. fixed.   You meant in the function call, right?

`(output_list,list)` -> `(output_list, list)`